extensions/ql-vscode/src/vscode-tests/no-workspace/remote-queries/variant-analysis-manager.test.ts

```diff
@@ -44,9 +44,11 @@ describe('Variant Analysis Manager', async function() {
     variantAnalysis = createMockApiResponse('in_progress', scannedRepos);
 
     try {
-      const extension = await extensions.getExtension<CodeQLExtensionInterface | Record<string, never>>('GitHub.vscode-codeql')!.activate();
-      cli = extension.cliServer;
-      variantAnalysisManager = new VariantAnalysisManager(extension.ctx, cli, storagePath, logger);
+      const ctx = createMockExtensionContext();
+      cli = createMockCliServer(sandbox);
+      const storagePath = path.join(__dirname);
```

Comment:
Do we need this storage path to be set to an existing path or can we just use an empty string instead? I don't think we're actually storing any results in these tests